routes/web.php

```diff
@@ -19,15 +20,27 @@
 Route::middleware('auth')->group(function () {
     Route::get('/', function () {
         return view('welcome');
-    });
+    })->name('dashboard');
+    Route::get('/registration/complete', [RegistrationController::class, 'registerComplete'])->name('register.complete');
```

Comment:
I think registration/complete route should be in the guest group because still not yet authenticated

Reply:
Hello, sir. There is a button in the register complete page which direct user to the dashboard.

Reply:
Ah I understand, I think it is good that complete page is not authenticated. Only authenticate when click to redirect to dashboard if possible?

Reply:
Ok, sir.